src/styles/App.css

```diff
@@ -0,0 +1,254 @@
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  color: black;
+  -webkit-font-smoothing: antialiased;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.home-header {
+  position: absolute;
+  height: 4rem;
+  width: 100%;
+  padding: 1.2em 2.2em;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #111;
+  font-size: 1rem;
+  font-weight: 700;
+  letter-spacing: 0.04rem;
+}
+
+.link {
+  text-decoration: underline;
+  color: #111;
+}
+
+.footer-link {
+  text-decoration: underline;
+}
+
+.body {
+  display: grid;
+  grid-template-columns: 0.2fr 0.8fr;
+  column-gap: 1.5em;
+  position: absolute;
+  width: 100vw;
+}
+
+/* Header for Docs page */
+
+.documentation-header {
+  display: flex;
+  height: 4rem;
+  width: 100%;
+  padding: 1.2em 2.2em;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #111;
+  font-size: 1rem;
+  font-weight: 700;
+  letter-spacing: 0.04rem;
+}
+
+.header-icon {
+  width: 2em;
+  height: 2em;
+  margin-right: 1.5em;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+/* SIDENAV */
+
+.sidenav {
+  display: flex;
+  flex-direction: column;
+  position: sticky;
+  height: 90vh;
+  background-color: white;
+  padding: 1em 2em 0em 2em;
+  margin-bottom: 2em;
+  border-radius: 0.5em;
+  box-shadow: 10px 10px 55px -8px rgba(0, 0, 0, 0.31);
+  overflow-y: scroll;
+  scroll-behavior: auto;
+  /* position: absolute; */
+  /* z-index: 10; */
+}
+
+.list {
+  padding-left: 0;
+}
+
+.sidenav-stacked-list {
+  text-align: left;
+  list-style: none;
+  margin: 2em 0;
+  /* padding: 0.5em 1.5em;
+  margin: 0.7em 0;
+  font-weight: 300;
+  align-items: center;
+  background-color: white; */
+}
+
+/* .sidenav-stacked-list:hover {
+  background-color: #1bc2e017;
+} */
+
+.sidenav-stacked-list-selected {
+  text-align: left;
+  align-items: center;
+  list-style: none;
+  margin: 0.7em 0;
+  /* padding: 0.5em 1.5em;
+  margin: 0.7em 0;
+  font-weight: 600;
+  color: #11a6c0;
+  background-color: #1bc2e017;
+  border-radius: 0.5em; */
+}
+
+.sidenav-navlink {
+  padding: 0.5em 1.5em;
+  margin: 0.7em 0;
+  font-weight: 300;
+  align-items: center;
+  background-color: white;
+}
+
+.sidenav-navlink:hover {
+  background-color: #1bc2e017;
+}
+
+.sidenav-navlink-selected {
+  padding: 0.5em 1.5em;
+  margin: 0.7em 0;
+  font-weight: 600;
+  color: #11a6c0;
+  background-color: #1bc2e017;
+  border-radius: 0.5em;
+}
+
+@media (max-width: 768px) {
+  .subtitle {
+    font-size: 10px;
+    padding: 1.2em 0em;
+  }
+  .logo {
+    font-size: 16px;
+  }
+
+  .header-icon {
+    width: 1.4em;
+    height: auto;
+  }
+
+  .body {
+    display: flex;
+    width: 100%;
+    left: 0;
+    right: 0;
+    justify-content: center;
+    height: auto;
+  }
+  .sidenav {
+    position: absolute;
+    z-index: 10;
```

Comment:
use a variable for z-indices too, helps to keep a track of all the indices we are using